install/resources/upgrade41.php

```diff
@@ -35,6 +35,21 @@ function upgrade41_dbchanges()
 	$db->insert_query('spiders', array("name" => "DuckDuckGo", "useragent" => "DuckDuckBot/1.1; (+http://duckduckgo.com/duckduckbot.html)"));
 	$db->insert_query('spiders', array("name" => "UptimeRobot", "useragent" => "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"));
 
-  	$output->print_contents("<p>Click next to continue with the upgrade process.</p>");
+	// Remove backslashes from last 1,000 log files.
+	$query = $db->simple_select('moderatorlog', '*', "action LIKE '%\\\\\\%", "limit" => 1000);
+
+	while($row = $db->fetch_array($query))
+	{
+    $original = $row['action'];
```

Comment:
Indentation here looks slightly off. My example code used 4 spaces, the core uses tabs.

Reply:
I'll fix that.